Src/BootCamp.Chapter/StringOps.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace BootCamp.Chapter
+{
+    public static class StringOps
+    {
+        public static bool IsStringValid(string message)
+        {
+            return !string.IsNullOrWhiteSpace(message) && !string.IsNullOrEmpty(message);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this condition so that it does not always evaluate to 'true'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5149394)